src/query/ast/src/ast/statements/copy.rs

```diff
@@ -117,6 +122,37 @@ impl CopyIntoTableStmt {
             CopyIntoTableOption::OnError(v) => self.on_error = v,
         }
     }
+
+    pub fn apply_to_copy_option(
+        &self,
+        copy_options: &mut CopyOptions,
+    ) -> common_exception::Result<()> {
+        copy_options.on_error =
+            OnErrorMode::from_str(&self.on_error).map_err(ErrorCode::SyntaxException)?;
+
+        if self.size_limit != 0 {
+            copy_options.size_limit = self.size_limit;
+        }
+
+        copy_options.split_size = self.split_size;
+        copy_options.purge = self.purge;
+        copy_options.disable_variant_check = self.disable_variant_check;
+        copy_options.return_failed_only = self.return_failed_only;
+
+        if self.max_files != 0 {
+            copy_options.max_files = self.max_files;
+        }
+
+        if !(copy_options.purge && self.force)
+            && copy_options.max_files > COPIED_FILES_MAX_COMMIT_NUM
+        {
+            return Err(ErrorCode::InvalidArgument(format!(
+                "max_files {} is too large, max_files should be less than {COPIED_FILES_MAX_COMMIT_NUM}",
```

Comment:
We can give a kindly error hints, tell user to copy with max_files option?

Reply:
here user is using `max_files`, but too large, maybe we can further suggest user to copy mutli times?.

`tell user to copy with max_files option` is already done when max_files=0 (no limit)

Reply:
> here user is using `max_files`, but too large, maybe we can further suggest user to copy mutli times?.

since user are already using  `max_files`, he should have known that.

Reply:
Yes，if the COPIED_FILES_MAX_COMMIT_NUM is 15000，if user COPY from a 150000 files stage, we can tell him you should COPY ... MAX_FILES=15000 many times.

Reply:
done